src/client/Meals.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { useState, useEffect } from "react";
+
+function Meals() {
+  const [meals, setMeals] = useState([]);
+  const [error, setError] = useState("");
+
+  useEffect(() => {
+    fetch(`http://localhost:8080/api/meals`)
```

Comment:
we have to fetch from /api/meals since we deployed  then we will not use the local data